packages/client/hmi-client/src/components/models/tera-model-diagram.vue

```diff
@@ -475,22 +481,42 @@ const contextMenuItems = ref([
 	}
 ]);
 
+const convertToIGraphHelper = (amr: Model) => {
+	if (isStratifiedAMR(amr)) {
+		// FIXME: wont' work for MIRA
+		return convertToIGraph(props.model?.semantics?.span?.[0].system);
```

Comment:
I see, this is why you were mentioning both MIRA and CatLab not having the same format matters.